docs/usage.md

```diff
@@ -62,10 +62,21 @@ the cloned repository.
 evg-module-manager disable --module wiredtiger
 ```
 
+## Performing git commit all
+
+Use the `git-commit` command to commit all your git tracked changes to your base repository and any
+modules that are currently enabled. The tool has already specified `-am` option to git, which means
```

Comment:
Instead of referring to these options as `-am` can we use their long forms (`--all` and `--message`) to make it clear what they mean?